packages/components/src/TableToolsTable/TableToolsTable.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Pagination, PaginationVariant } from '@patternfly/react-core';
+import { Table, TableBody, TableHeader } from '@patternfly/react-table';
+import PrimaryToolbar from '../PrimaryToolbar';
+import TableToolbar from '../TableToolbar';
+import useTableTools from '@redhat-cloud-services/frontend-components-utilities/useTableTools';
+
+const TableToolsTable = ({ items = [], columns = [], filters = [], options = {}, toolbarProps: toolbarPropsProp, ...tablePropsRest }) => {
```

Comment:
Added all of them.

Reply:
I thought about this again, and I think I'll remove them again. I think of this Table component being sortof default table for the platform and there should be no need to extensively customise it. If there is a new feature or change to be made to any table it should be made here rather than as a customisation, to mitigate the chance of inconsistencies across applications. I will also look into ways of not needing  the custom toolbar and table props, these were added as shortcuts/workarounds and are reason for a few not so nice parts in the code to deal with them. 

Should there be a reason for a different component body, for example one that doesn't have a PrimaryToolbar or maybe one migrating to the new PF composable tables, only the `useTableTools` hook can be used to invoke all hooks at once or the underlying hooks can also be used individually with Patternfly (based) Table components, for example the Compliance `SystemsTable` only uses a few hooks, like the `useFilterConfig` hook and `useBulkSelect` hook with the `InventoryTable`